operator-common/src/main/java/io/strimzi/operator/common/Annotations.java

```diff
@@ -18,6 +18,7 @@
 
     public static final String STRIMZI_DOMAIN = "strimzi.io/";
     public static final String STRIMZI_LOGGING_ANNOTATION = STRIMZI_DOMAIN + "logging";
+    public static final String STRIMZI_LOGGING_APPENDERS_ANNOTATION = STRIMZI_DOMAIN + "logging-appenders";
```

Comment:
How does this differ from `ANNO_STRIMZI_LOGGING_APPENDERS_HASH` from #3469 ? Is it really different annotation - i.e. not hash? If yes, why are we using hash there and not here? I guess we need to keep the old values and be able to recover them?